packages/components/addon/components/hds/disclosure/index.js

```diff
@@ -7,41 +7,47 @@ export default class HdsDisclosureComponent extends Component {
   @tracked toggleRef;
   @tracked isToggleClicked;
 
+  @action
+  didInsert(element) {
+    this.element = element;
+  }
+
   @action
   onClickToggle(event) {
     // we store a reference to the DOM node that has the "onClickToggle" event associated with it
     if (!this.toggleRef) {
       this.toggleRef = event.currentTarget;
     }
     this.isActive = !this.isActive;
+    this.toggleRef.focus();
```

Comment:
[suggestion] generally, I tend to add comments to this kind of "things" (when I make a non-obvious fix for two reasons: a future me that will come in and don't remember why I did something + the fear that a future maintainer will come in and remove the code because they think is unnecessary).